### PR TITLE
Fix navigation menu on mobile

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 

applied @helen-dikareva 's fix
pls check once again on mobile

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
it still performs strange in IPad emulator and sometimes in landscape - check it pls
@helen-dikareva

File: src/_includes/head.html [hunk 1]
```diff
@@ -54,57 +53,82 @@
...
-            var FOOTER_HEIGHT  = 63;
-            var HEADER_HEIGHT  = 52;
+        var MIN_BODY_WIDTH = 900;
+        var FOOTER_HEIGHT  = 63;
+        var HEADER_HEIGHT  = 52;
 
-            function updatePosition() {
-                var footerVisibleHeight = FOOTER_HEIGHT - document.body.scrollHeight + window.pageYOffset + document.body.clientHeight;
+        var nav            = null;
+        var isMobileDevice = window.screen.width < 800 || window.screen.height < 800;
```
Comment (VasilyStrelyaev) [inline src/_includes/head.html]: @helen-dikareva I've increased the resolution to 800 here but obviously it's still larger on modern devices.
Anyway, I can't see a better workaround.

**Comment by VasilyStrelyaev** [comment]
Let's merge it before we update the website for v0.10.0.
After all, it fixes the most evident issues.
@helen-dikareva take the last look please...

**Comment by helen-dikareva** [comment]
discussed in private

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
Would anybody else review?

**Pull request merged**